### ISSUE TITLE
[LayoutLM ] ,SROIE dataset details

### ISSUE THREAD

**Issue by 106279928** [microsoft/unilm#168]
Thanks for your wanderful work !
     I am trying the model on SROIE dataset，but I found some details  is not clear .I have saw this issue and anwser：#151 ,but I have another quesion.
in  issue #151: there has two choice 
       choice A.
         BOOK TA .K (TAMAN DAYA) SDN BHD S-COMPANY

       choice B.
           BOOK S-COMPANY
           TA .K S-COMPANY
           (TAMAN DAYA) S-COMPANY
and your advice is choice B . 
     
but as you konw , the token is wordpiced, so the dataset would preprocess like shown the below ?
(tokenized and labeled)
           book    B-COMPANY
           ta		I-COMPANY
           .		I-COMPANY
           k		I-COMPANY
          (		I-COMPANY
          tam	I-COMPANY
          ##an	I-COMPANY
          day	I-COMPANY
          #a		I-COMPANY
          )		I-COMPANY
        sd		I-COMPANY
        ##n		I-COMPANY
        b		I-COMPANY
       ##hd	 E-COMPANY

thanks !!

**Comment by renjithsasidharan** [comment]
If you look at the code [here](https://github.com/microsoft/unilm/blob/master/layoutlm/layoutlm/data/funsd.py#L254), you can see that sub-tokens are labelled with padding token.

```python
word_tokens = tokenizer.tokenize(word)
            tokens.extend(word_tokens)
            token_boxes.extend([box] * len(word_tokens))
            actual_bboxes.extend([actual_bbox] * len(word_tokens))
            # Use the real label id for the first token of the word, and padding ids for the remaining tokens
            label_ids.extend(
                [label_map[label]] + [pad_token_label_id] * (len(word_tokens) - 1)
            )
```

**Comment by 106279928** [comment]
thanks , I have seen this code ,but I want to build the dataset ，so I need more details :
1、Whether to use only spaces to split words ？
2、Label method is using BIE ,or BI ？ 
3、Whether the text lines need to be reorganized according to coordinates？
4、should I give the coordinates for each words？or just use the boxing value of lines ？

**Comment by yangjinning2020** [comment]
Can you run the code successfully? In LayoutLM source code, it uses -100 as pad_token_label_id.  I met error when I run the model. It shows "ClassNLLCriterion.cu:106: block: [0,0,0], thread: [0,0,0] Assertion `t >= 0 && t < n_classes` failed."

**Comment by sungraepark** [comment]
Do you have a plan to share the word-level (box-level) annotations of SROIE? It can be helpful in this field. If not, all researchers might make different annotations and the results from them cannot be compared.

**Comment by ruifcruz** [comment]
yes please share at least a sample of it, it would be of great help

**Comment by ruifcruz** [comment]
Mabe it helps: https://github.com/ruifcruz/sroie-on-layoutlm
